plugins/jobs/src/js/pages/JobConfiguration.js

```diff
@@ -0,0 +1,139 @@
+          <ConfigurationMapLabel>Memory (MiB)</ConfigurationMapLabel>
+          <ConfigurationMapValue>{job.mem}</ConfigurationMapValue>
+        </ConfigurationMapRow>
+        <ConfigurationMapRow>
+          <ConfigurationMapLabel>Disk Space (Mib)</ConfigurationMapLabel>
+          <ConfigurationMapValue>{job.disk}</ConfigurationMapValue>
+        </ConfigurationMapRow>
+        <ConfigurationMapRow>
+          <ConfigurationMapLabel>Command</ConfigurationMapLabel>
+          <ConfigurationMapValue>
+            <pre className="flush transparent wrap">{job.command}</pre>
+          </ConfigurationMapValue>
+        </ConfigurationMapRow>
+      </ConfigurationMapSection>
+    );
+  }
+
+  getScheduleSection(job) {
+    const [schedule] = job.schedules.nodes;
+    if (schedule == null) {
+      return null;
+    }
+
+    return (
+      <ConfigurationMapSection>
+        <ConfigurationMapHeading>Schedule</ConfigurationMapHeading>
+        <ConfigurationMapRow>
+          <ConfigurationMapLabel>ID</ConfigurationMapLabel>
+          <ConfigurationMapValue>{schedule.id}</ConfigurationMapValue>
+        </ConfigurationMapRow>
+        <ConfigurationMapRow>
+          <ConfigurationMapLabel>Enabled</ConfigurationMapLabel>
+          <ConfigurationMapValue>{schedule.enabled}</ConfigurationMapValue>
+        </ConfigurationMapRow>
+        <ConfigurationMapRow>
+          <ConfigurationMapLabel>CRON Schedule</ConfigurationMapLabel>
+          <ConfigurationMapValue>{schedule.cron}</ConfigurationMapValue>
+        </ConfigurationMapRow>
+        <ConfigurationMapRow>
+          <ConfigurationMapLabel>Time Zone</ConfigurationMapLabel>
+          <ConfigurationMapValue>{schedule.timezone}</ConfigurationMapValue>
+        </ConfigurationMapRow>
+        <ConfigurationMapRow>
+          <ConfigurationMapLabel>Starting Deadline</ConfigurationMapLabel>
+          <ConfigurationMapValue>
+            {schedule.startingDeadlineSeconds}
+          </ConfigurationMapValue>
+        </ConfigurationMapRow>
+      </ConfigurationMapSection>
+    );
+  }
+
+  getDockerContainerSection(job) {
+    const docker = job.docker;
+    if (docker == null || !docker.image) {
+      return null;
+    }
+
+    return (
+      <ConfigurationMapSection>
+        <ConfigurationMapHeading>Docker Container</ConfigurationMapHeading>
+        <ConfigurationMapRow>
+          <ConfigurationMapLabel>Image</ConfigurationMapLabel>
+          <ConfigurationMapValue>{docker.image}</ConfigurationMapValue>
+        </ConfigurationMapRow>
+      </ConfigurationMapSection>
+    );
+  }
+
+  getLabelSection(job) {
+    return (
+      <HashMapDisplay
+        hash={job.labels.reduce(
+          (memo, { key, value }) => ({ ...memo, [key]: value }),
+          {}
+        )}
+        headline="Labels"
+      />
+    );
+  }
+
+  render() {
+    const { job } = this.props;
+
+    return (
+      <div className="container">
+        <ConfigurationMap>
+          {this.getGeneralSection(job)}
```

Comment:
🎨 I think this is legacy code, but personally I dont like these `this.getSomething()` things… why not just create a new `<GeneralSection />` component?

Reply:
done 👍